src/zope/app/onlinehelp/onlinehelptopic.py

```diff
@@ -153,20 +158,24 @@ def __init__(self, id, title, path, parentPath, interface=None, view=None):
 
         super(BaseOnlineHelpTopic, self).__init__()
 
+    def _newContainerData(self):
+        # Ensure consistent iteration order for tests.
+        from collections import OrderedDict
+        return OrderedDict()
```

Comment:
I suspect that's probably more of a problem in zope.security's default allow lists, though.